src/lib/netlist/core/base_objects.h

```diff
@@ -21,10 +21,7 @@
 
 #include <unordered_map>
 
-namespace netlist
-{
-
-	namespace detail {
+namespace netlist::detail {
 
 		template <typename C, typename T>
```

Comment:
This has an extra level of indent now that you’re using a nested namespace definition.

Reply:
I wanted to fix indentation in a separate step. Changed now.